riscv/src/compiler.rs

```diff
@@ -383,6 +383,12 @@ instr is_positive X -> Y {
     Y = wrap_bit
 }
 
+instr poseidon Y, Z -> X {
+    // Dummy code, to be replaced with actual poseidon code.
+    X = Y,
+    X = Z
```

Comment:
This is obviously a placeholder, but a very dumb one that doesn't let me check that everything works past the witness generation stage. Some input on how to write a better dummy would be appreciated.

For reference, this is what I get right now:

```
Error: Row 643: Identity check failed or unable to derive values for some witness columns.
Set RUST_LOG=debug for more information.
thread 'main' panicked at 'Witness generation failed.', /home/gballet/src/powdr/executor/src/witgen/generator.rs:189:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```